bdd_tests/features/cli_flags.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Check command line options provided by CCX Notification Writer
+
+
+  Scenario: Check if CCX Notification Writer displays help message
+    Given the system is in default state
+     When I start the CCX Notification Writer with the --help command line flag
+     Then I should see help messages displayed on standard output
+
+
+  Scenario: Check if CCX Notification Writer displays version info
+    Given the system is in default state
+     When I start the CCX Notification Writer with the --version command line flag
+     Then I should see version info displayed on standard output
+
+
+  Scenario: Check if CCX Notification Writer displays authors
+    Given the system is in default state
+     When I start the CCX Notification Writer with the --authors command line flag
+     Then I should see info about authors displayed on standard output
+
+
+  Scenario: Check the ability to drop initialize all database tables
+    Given Postgres is running
+      And CCX Notification Writer database is created for user postgres with password postgres
```

Comment:
I would add "And CCX Notification Writer database is not set up" condition, which would check that the tables do not exist in the DB.

Reply:
will do, thank you

Reply:
Implemented via new commit